priv/perf/apps/load_test/lib/scenario/childchain_submit_transactions.ex

```diff
@@ -0,0 +1,87 @@
+# Copyright 2019-2020 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule LoadTest.Scenario.ChildChainSubmitTransactions do
+  @moduledoc """
+  This scenario tests childchain handling lots of transactions concurrently
+  """
+
+  use Chaperon.Scenario
+
+  alias Chaperon.Timing
+  alias ExPlasma.Utxo
+  alias LoadTest.Service.Faucet
+  alias LoadTest.Ethereum.Account
+
+  @eth <<0::160>>
+
+  @spec run(Chaperon.Session.t()) :: Chaperon.Session.t()
+  def run(session) do
+    fee_wei = Application.fetch_env!(:load_test, :fee_wei)
+
+    # Create a new account and fund it from the faucet
+    {:ok, sender} = LoadTest.Ethereum.Account.new()
+    ntx_to_send = config(session, [:ntx_to_send])
+    initial_funds = ntx_to_send + ntx_to_send * fee_wei
+    {:ok, utxo} = Faucet.fund_child_chain_account(sender, initial_funds, @eth)
+
+    Logger.debug("User utxo: #{inspect(utxo)}")
```

Comment:
hmmm probably use the session log_debug instead

Reply:
Ah, that was just debug logging 😄